src/util.cpp

```diff
@@ -7,6 +7,11 @@
 #include "config/bitcoin-config.h"
 #endif
 
+#if (defined(__FreeBSD__) || defined(__OpenBSD__))
```

Comment:
IMHO this should be moved below ``#include <stdarg.h>`` and also take into account what we have there Win/Linux def/ifdef etc. (I didn't check that).